nip47/event_handler_test.go

```diff
@@ -379,30 +525,35 @@ func TestHandleResponse_IncorrectPubkey(t *testing.T) {
 	assert.NoError(t, err)
 
 	content := map[string]interface{}{
-		"method": models.GET_BALANCE_METHOD,
+		"method": models.GET_INFO_METHOD,
 	}
 
 	payloadBytes, err := json.Marshal(content)
 	assert.NoError(t, err)
 
-	msg, err := nip04.Encrypt(string(payloadBytes), ss)
+	msg, err := cipher.Encrypt(string(payloadBytes))
 	assert.NoError(t, err)
 
+	// don't pass correct version
 	reqEvent := &nostr.Event{
 		Kind:      models.REQUEST_KIND,
+		PubKey:    reqPubkey,
 		CreatedAt: nostr.Now(),
-		Tags:      nostr.Tags{},
+		Tags:      nostr.Tags{[]string{"v", requestVersion}},
 		Content:   msg,
 	}
-	err = reqEvent.Sign(reqPrivateKey2)
-	assert.NoError(t, err)
 
-	// set a different pubkey (this will not pass validation)
-	reqEvent.PubKey = reqPubkey
+	if requestVersion != "0.0" {
```

Comment:
This looks wrong, and duplicated

Reply:
Whoops sorry, might have missed while changing, what would you prefer though? Passing version tag everytime reduces a conditional but then we don't check the case where version is not specified, should we pass version all the time and have a separate test where we check if a request event without any version tag is considered as "0.0"?

Reply:
I guess we could do `if requestVersion != "" {` and then pass `""` for the third test. And instead of removing the line on this comment, we don't set the tags a few lines above

Reply:
> have a separate test where we check if a request event without any version tag is considered as "0.0"?
I think this would be good too

Reply:
> I guess we could do if `requestVersion != "" {` and then pass `""` for the third test

I like this better since then we can check this for all other tests as well, what should we name it though 🤔 

`_Nip04_Unspecified`?